src/test/forms/cashflow/CashFlowInputsRow.test.js

```diff
@@ -1,63 +1,75 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { mount, shallow } from 'enzyme';
```

Comment:
Out of curiosity, why not make the last test into `mount()` too?

Reply:
All I remember is that it was hard to resolve the strangeness that was happening. I'm not super familiar with enzyme, though, so maybe it's really a quick fix. I encourage you to try it out!